engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -131,9 +133,13 @@ public WorldRendererLwjgl(WorldProvider worldProvider, ChunkProvider chunkProvid
         }
         localPlayerSystem.setPlayerCamera(playerCamera);
 
+        initMainDirectionalLight();
+
         renderableWorld = new RenderableWorldImpl(worldProvider, chunkProvider, bufferPool, playerCamera, shadowMapCamera);
         renderQueues = renderableWorld.getRenderQueues();
+    }
 
+    private void initMainDirectionalLight() {
         mainDirectionalLight.lightType = LightComponent.LightType.DIRECTIONAL;
```

Comment:
Should I convert this object into a POJO instead of using a component? In the long term I'd like the renderer not to have any preferential main directional light and be able to handle, say, multiple celestial light sources, hence having lights provided by the sky sphere. But for the time being, if using an unattached LightComponent is inappropriate, I could make this a simple class.